seed/challenges/02-data-visualization-certification/sass.json

```diff
@@ -1,19 +1,307 @@
+        "",
+        "So what's the point of using one?",
+        "CSS preprocessors such as <code>Sass</code> or <code>Less</code> allow you to do nifty things that regular CSS can't do. They allow you to create <code>variables</code>, write more compact code with <code>nesting</code>, <code>import</code> other CSS files and much more.",
+        "",
+        "In this series of Waypoints, we will be teaching you the basics of <code>Sass</code>"
+      ],
+      "challengeSeed": [],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690ad185b3908042c47b328",
+      "title": "Storing Data with Sass Variables",
+      "description": [
+        "Just like in Javascript, you can declare variables to store data in. ",
+        "In Javascript, you would use the <code>var</code> keyword to define a variable. When decaring variables in <code>Sass</code>, you start by using a <code>$</code> followed by a string.",
+        "Here is an example: <code>$color = green;</code>",
+        "Making variables become exteremly useful when we want similar elemnts to have the same color, for example. If wanted to change the color to maybe a slightly different shade, all we would have to do is change the variables value.",
+        "<h4>Instructions</h4>",
+        "Create a variable <code>$text-color</code> and set it equal to <code>red</code>",
+        "Then change the value of all <code>color</code> properties by using <code>color: $text-color</code>.",
+        "Finally, change the value of <code>$text-color</code> to another color and see how all respective elements change color!"
+      ],
+      "challengeSeed": [
+        "<style>",
+        "  /*Only change code below this line.*/",
+        "  ",
+        "  /*Only change code below this line.*/",
+        "  .header{",
+        "    text-align: center;",
+        "  }",
+        "  .blog-post h3{",
+        "  /*Only change code below this line.*/",
+        "    color: red;",
+        "  /*Only change code below this line.*/",
+        "  }",
+        "</style>",
+        "",
+        "<h1 class=\"header\">Lern Sass</h1>",
+        "<div class=\"blog-post\">",
+        "  <h3>Some random title</h3>",
+        "  <p>This is a pargraph with some very random text in it</p>",
+        "</div>",
+        "<div class=\"blog-post\">",
+        "  <h3>Header #2</h3>",
+        "  <p>Here is some more random text.</p>",
+        "</div>",
+        "<div class=\"blog-post\">",
+        "  <h3>Here is another header</h3>",
+        "  <p>Even more random text within a paragraph</p>",
+        "</div>"
+      ],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690af775b3908042c47b329",
+      "title": "Nesting your CSS",
+      "description": [
+        "<code>Nesting</code> is a really useful way of organising your CSS.",
```

Comment:
Are we going to use the North American spelling of "organizing"? 😆  cc/ @atjonathan